README.md

```diff
@@ -45,6 +45,12 @@ There are simple in memory implementations of an event store and entity repo. Th
 ### MongoDB
 
 Fairly mature, used in production.
+_Warning: Default MongoDB eventstore could lead to MongoDB errors while saving aggregates exceeding 16MB in size._
```

Comment:
Would be nicer as “Note: This implementation has a document size limit of 16Mb, which could lead to errors for large aggregates. If needed use the alternative version below.”